solc/CommandLineInterface.cpp

```diff
@@ -306,9 +306,35 @@ void CommandLineInterface::handleFormal()
 		cout << "Formal version:" << endl << m_compiler->formalTranslation() << endl;
 }
 
-void CommandLineInterface::readInputFilesAndConfigureRemappings()
+void CommandLineInterface::readInputFilesAndConfigureRemappings(bool usestdin, vector<string> inputFiles)
```

Comment:
Arguments should have `_` as prefix and vectors should be passed by const reference.